packages/salesforcedx-vscode-core/package.nls.json

```diff
@@ -1,74 +1,50 @@
 {
   "running_tasks_title_text": "Running Tasks",
-
   "force_auth_web_login_authorize_dev_hub_text": "SFDX: Authorize a Dev Hub",
+  "force_auth_web_login_authorize_org_text": "SFDX: Authorize an Org",
```

Comment:
This line is the only change that I made to this file. The rest of the changes are a result of automatic formatting. Should I undo those changes?

Reply:
Are you using prettier to format? Did you configure any other options for prettier? The only option we have is to use single quotes instead of double.

Reply:
It appears that your default file length for prettier might be longer than the default 80 that we use.